starkinfra/individualdocument/__individualdocument.py

```diff
@@ -0,0 +1,122 @@
+from ..utils import rest
+from starkcore.utils.resource import Resource
+from starkcore.utils.checks import check_datetime, check_date
+
+
+class IndividualDocument(Resource):
+    """# IndividualDocument object
+    IndividualDocuments is an images part of document faces (front face, back face)
```

Comment:
Individual documents are images containing the face of a document and a selfie to be used in a matching validation. When created they attached to an individual identity to be used for its validation.